flexmeasures/data/schemas/reporting/__init__.py

```diff
@@ -0,0 +1,50 @@
+from marshmallow import Schema, fields, validate
+
+from flexmeasures.data.schemas.sensors import SensorIdField
+from flexmeasures.data.schemas.sources import DataSourceIdField
+
+from flexmeasures.data.schemas import AwareDateTimeField, DurationField
+
+
+class TimeBeliefQueryConfigSchema(Schema):
+    """
+    This schema implements the required fields to perform a TimeBeliefs search
+    using the method flexmeasures.data.models.time_series:TimedBelief.search
```

Comment:
We are now using the method `flexmeasures.data.models.time_series:Sensor.search_beliefs`.